R/pkgdown.R

```diff
@@ -154,6 +168,21 @@ pkgdown_config_path <- function() {
   )
 }
 
+has_logo <- function() {
+  !is.null(
+    path_first_existing(c("man/figures/logo.png", "man/figures/logo.svg"))
+  )
+}
+
+potentially_has_logo <- function() {
```

Comment:
This idea came to me, because of PRs I opened in various repos of package devs who didn't seem to be aware of this. see #1995